src/modules/fancyzones/FancyZonesLib/FancyZonesLib.vcxproj

```diff
@@ -67,6 +68,10 @@
     <ClInclude Include="ZonesOverlay.h" />
   </ItemGroup>
   <ItemGroup>
+    <ClCompile Include="FancyZonesData\CustomLayouts.cpp">
```

Comment:
this looks a bit weird. Does it have to be this way?

Reply:
Otherwise, we'll have a pipeline build fail. See https://github.com/microsoft/PowerToys/pull/15624